suite-native/module-home/src/screens/HomeScreen/components/PortfolioGraphHeader.tsx

```diff
@@ -10,10 +10,12 @@ import {
     referencePointAtom,
     selectedPointAtom,
 } from '../portfolioGraphAtoms';
+import { selectHasDeviceHistoryEnabledAccounts } from '@suite-native/graph/src/selectors';
 
 export const PortfolioGraphHeader = () => {
```

Comment:
If you will be renanimg the AccountDetailGraphHeader, please rename this component too.

Reply:
10127190bd